src/studio/kdb/Server.java

```diff
@@ -147,12 +147,12 @@ public String toString() {
         return getFullName();
     }
 
-    public String getConnectionString(boolean includeCreditional) {
-        String connection = "`:" + host + ":" + port;
-        if (!includeCreditional) return connection;
-
-        return connection + ":" + username + ":" + password;
+    public String getConnectionString() {
+        return "`:" + host + ":" + port;
+    }
 
+    public String getConnectionStringWithPw() {
```

Comment:
May be it's better to name `getConnectionStringWithPwd` ? I think "pwd" has more associations with password than "pw".